crates/wasm/src/compile.rs

```diff
@@ -95,19 +95,7 @@ pub fn compile(args: JsValue) -> JsValue {
             .compile_contracts(&options.compile_options)
             .unwrap_or_else(|_| panic!("Contract compilation failed"));
 
-        // Flatten each contract into a list of its functions, each being assigned a unique name.
-        let collected_compiled_programs: Vec<_> = compiled_contracts
```

Comment:
Can you test what happens if you try to compile the same contract using the CLI?